src/storage/checkpoint_io.cpp

```diff
@@ -0,0 +1,60 @@
+#include "storage/checkpoint_io.h"
+#include <vector>
+namespace terrier::storage {
+
+void BufferedTupleWriter::SerializeTuple(ProjectedColumns::RowView *row, ProjectedRow *row_buffer,
+                                         const storage::BlockLayout &layout) {
+  // First construct tuple in row_buffer, and calculate total size required
+  int32_t varlen_offset = 0;
+  std::vector<const VarlenEntry *> varlen_entries;
+  for (uint16_t projection_list_idx = 0; projection_list_idx < row->NumColumns(); projection_list_idx++) {
+    if (row->IsNull(projection_list_idx)) {
+      row_buffer->SetNull(projection_list_idx);
+    } else {
+      row_buffer->SetNotNull(projection_list_idx);
```

Comment:
AccessForceNotNull already sets not null

Reply:
resolved in the latest version